### ISSUE TITLE
API Credential Category

### ISSUE THREAD

**Issue by Christianuss** [RohanNagar/op-connect-sdk-java#4]
```
com.fasterxml.jackson.databind.exc.InvalidFormatException: Cannot deserialize value of type `com.sanctionco.opconnect.model.Category` from String "API_CREDENTIAL": not one of the values accepted for Enum class: [SERVER, SOFTWARE_LICENSE, WIRELESS_ROUTER, CUSTOM, PASSPORT, REWARD_PROGRAM, SOCIAL_SECURITY_NUMBER, SECURE_NOTE, PASSWORD, OUTDOOR_LICENSE, LOGIN, DRIVER_LICENSE, CREDIT_CARD, DATABASE, IDENTITY, EMAIL_ACCOUNT, MEMBERSHIP, BANK_ACCOUNT, DOCUMENT]
```
When attempting to list items in a vault that has items of the API Credential type - I believe the Category enum is missing `API_CREDENTIAL`.

**Comment by RohanNagar** [comment]
Thanks for filing this bug - I will fix this by tomorrow.

**Comment by RohanNagar** [comment]
Interestingly I can't seem to reproduce this.

I created an API Credential Item:

<img width="416" alt="Screen Shot 2021-05-03 at 8 17 12 AM" src="https://user-images.githubusercontent.com/5522930/116895364-fae46800-abe7-11eb-8f7e-2610e4b99f78.png">

and then with the current codebase, it works and returns type `CUSTOM`.

```
Listing items that contain 'API' in the title...
Got 1 items:[Item[id='waqx6e4okqw63xte2ak5n4xwna', title='My New API', vault=VaultId[id='exlkgr6ahvgdvhxrudvndtez7e'], category=CUSTOM, urls=[], favorite=false, tags=[], version=1, trashed=false, createdAt=2021-05-03T15:07:49Z, updatedAt=2021-05-03T15:08:17Z, lastEditedBy='5QY7LUBSRZA5RHHP3Q66QSTSJE', sections=[], fields=[]]]
Getting details of the first item...
Got item: Item[id='waqx6e4okqw63xte2ak5n4xwna', title='My New API', vault=VaultId[id='exlkgr6ahvgdvhxrudvndtez7e'], category=CUSTOM, urls=[], favorite=false, tags=[], version=1, trashed=false, createdAt=2021-05-03T15:07:49Z, updatedAt=2021-05-03T15:08:17Z, lastEditedBy='5QY7LUBSRZA5RHHP3Q66QSTSJE', sections=[], fields=[Field[id='notesPlain', purpose=NOTES, type=STRING, label='notesPlain', value='null', generate=null, recipe=null, section=null, entropy=null], Field[id='username', purpose=null, type=STRING, label='username', value='Tester', generate=null, recipe=null, section=null, entropy=null], Field[id='credential', purpose=null, type=CONCEALED, label='credential', value='testtesttest', generate=null, recipe=null, section=null, entropy=null], Field[id='type', purpose=null, type=MENU, label='type', value='bearer', generate=null, recipe=null, section=null, entropy=null], Field[id='filename', purpose=null, type=STRING, label='filename', value='null', generate=null, recipe=null, section=null, entropy=null], Field[id='validFrom', purpose=null, type=DATE, label='valid from', value='null', generate=null, recipe=null, section=null, entropy=null], Field[id='expires', purpose=null, type=DATE, label='expires', value='null', generate=null, recipe=null, section=null, entropy=null], Field[id='hostname', purpose=null, type=STRING, label='hostname', value='null', generate=null, recipe=null, section=null, entropy=null]]]
```

Even when I add the `API_CREDENTIAL` category to the enum it still comes back as `CUSTOM` for me.

**Comment by RohanNagar** [comment]
Ah never mind, I figured it out. I needed to pull the latest version of the connect-sync/connect-api docker images. It fails now without the `API_CREDENTIAL` category.

**Comment by RohanNagar** [comment]
Fixed in d9c44b2
